.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+* [ ] The commit message follows our guidelines: https://github.com/vmware/clarity/blob/master/CONTRIBUTING.md
```

Comment:
LGTM. I think we probably need to have a certain commit convention or guideline now since we don't have one in the `CONTRIBUTING.md` file

Reply:
See updated changes about using https://www.conventionalcommits.org pre commit hook (based on the Angular style)